syntax = "proto3";
package PB;

message C_CHAT {
    string msg = 1;
}

message S_CHAT {
    uint64 playerId = 1;
    string msg = 2;
}
